build/lib/topostats/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""Topostats"""
```

Comment:
Pulled in a bunch of `build/lib/*` files again.

Feel free to add the following to `.gitignore` once these are removed, then you and no one else will pull them in again.

```
build/*
```

Reply:
Cool, done!